<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xmlExample</class>
 <widget class="QMainWindow" name="xmlExample">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1889</width>
    <height>1053</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>xmlExample</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ptn_read">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ptn_write">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_tips">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>980</y>
      <width>1191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>70</y>
      <width>751</width>
      <height>791</height>
     </rect>
    </property>
    <property name="title">
     <string>操作区</string>
    </property>
    <widget class="QTreeWidget" name="menu_tree">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>671</width>
       <height>571</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="ptn_delete">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>690</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ptn_movedowm">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>690</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>下移</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ptn_moveup">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>690</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>上移</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ptn_pase_attr">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>690</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>粘贴</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ptn_insert">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>690</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>插入</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_Frame">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>311</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Frame</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ptn_copy">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>690</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>复制</string>
     </property>
    </widget>
    <zorder>ptn_delete</zorder>
    <zorder>groupBox_2</zorder>
    <zorder>menu_tree</zorder>
    <zorder>ptn_movedowm</zorder>
    <zorder>label</zorder>
    <zorder>cb_Frame</zorder>
    <zorder>ptn_insert</zorder>
    <zorder>ptn_moveup</zorder>
    <zorder>cb_all</zorder>
    <zorder>cb_text</zorder>
    <zorder>groupBox_3</zorder>
    <zorder>ptn_pase_attr</zorder>
    <zorder>ptn_setitem</zorder>
    <zorder>cb_advanced</zorder>
    <zorder>label_5</zorder>
    <zorder>le_name</zorder>
    <zorder>groupBox</zorder>
    <zorder>cb_display</zorder>
    <zorder>ptn_delete</zorder>
    <zorder>ptn_movedowm</zorder>
    <zorder>menu_tree</zorder>
    <zorder>ptn_moveup</zorder>
    <zorder>cb_Frame</zorder>
    <zorder>label</zorder>
    <zorder>ptn_pase_attr</zorder>
    <zorder>ptn_insert</zorder>
    <zorder>menu_tree</zorder>
    <zorder>ptn_delete</zorder>
    <zorder>ptn_movedowm</zorder>
    <zorder>ptn_moveup</zorder>
    <zorder>ptn_pase_attr</zorder>
    <zorder>ptn_insert</zorder>
    <zorder>cb_Frame</zorder>
    <zorder>label</zorder>
    <zorder>ptn_copy</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>491</width>
      <height>791</height>
     </rect>
    </property>
    <property name="title">
     <string>属性</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>421</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>Advanced</string>
     </property>
     <widget class="QLineEdit" name="le_clone">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbn_clone">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>clone</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbn_color">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>191</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>color background</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbn_bitmap">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>191</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>bitmap background</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>361</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Normal,         focus,          Disable</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_anomcolor">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_afoccolor">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_adiscolor">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>120</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>61</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Normal     

focus

Disable</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nomBitmap">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_focBitmap">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_disBitmap">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>260</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="cb_all">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>复制所有</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_name">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>421</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>text</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>391</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Align,     NormColor,   FocColor,   DisColor</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_disColor">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_Align">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_NormColor">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_FocColor">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_advanced">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>21</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_display">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>21</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_text">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>21</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ptn_setitem">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>690</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>修改属性</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>421</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Display</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Left,    Top,    Width,    Height</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_XPos">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_YPos">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_Width">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_Height">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>70</y>
      <width>491</width>
      <height>791</height>
     </rect>
    </property>
    <property name="title">
     <string>图片</string>
    </property>
    <widget class="QListView" name="lv_img_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>471</width>
       <height>481</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ptn_addimg">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>添加图片</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_img">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>570</y>
       <width>451</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string>图像</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1889</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
